src/js/options.js

```diff
@@ -258,7 +321,23 @@ var SettingsTab = {
       await LocalStore.setOne(StoreKey.SETTINGS, newSettings);
 
       // Update context menu
-      mainConfigurationUpdated(true);
+      mainConfigurationUpdated();
+    }
+  },
+
+  onDropdownSelect: async function (event) {
+    console.log(event);
+    event.preventDefault();
+    var targetName = _.get(event, "target.name");
+
+    if (!_.isEmpty(targetName)) {
+      var newSettings =
+        _.clone(await LocalStore.getOne(StoreKey.SETTINGS)) || {};
+      newSettings[targetName] = event.target.value;
+      await LocalStore.setOne(StoreKey.SETTINGS, newSettings);
+
+      // Update The UI
+      SettingsTab.updateForms();
```

Comment:
Same as above, why we must re-render all forms on the Settings tabs after a dropdown is changed?